% \documentclass[../main.tex]{subfiles}

% \begin{document}

{
\chapter*{\vspace{-3cm}Appendix}
\addcontentsline{toc}{chapter}{Appendix}
\label{appendix}
\markboth{Appendix}{Appendix}
}

The appendix includes the titles and abstracts of the papers published during my PhD that are not part of this thesis.

\clearpage

%----------------------------------------------------------------------------------

{
\setstretch{0.8}
\section*{\LARGE{Taxonomic revision of the Australian stick insect genus \textit{Candovia} (Phasmida: Necrosciinae): insight from molecular systematics and species-delimitation approaches.}}

\vspace{4mm}

\noindent{\large{Giobbe Forni\textsuperscript{1,2}, Alex Cussigh\textsuperscript{1,2}, Paul D. Brock\textsuperscript{3}, Braxton R. Jones\textsuperscript{4}, Filippo Nicolini\textsuperscript{1}, Jacopo Martelossi\textsuperscript{1}, Andrea Luchetti\textsuperscript{1}}, Barbara \\ Mantovani\textsuperscript{1}}

\vspace{4mm}

\noindent{\textsuperscript{1}\textit{Department of Biological, Geological and Environmental Sciences, University of Bologna, Bologna, Italy}.}

\noindent{\textsuperscript{2}\textit{Department of Agricultural and Environmental Sciences, University of Milan, Milano, Italy}.}

\noindent{\textsuperscript{3}\textit{The Natural History Museum, Cromwell Road, London, UK}.}

\noindent{\textsuperscript{4}\textit{School of Life and Environmental Sciences, The University of Sydney, Sydney NSW 2006, Australia}.}

\vspace{4mm}

\noindent{\textbf{Published in}: 2023, \textit{Zoological Journal of the Linnean Society}, 197:189--210. \href{https://doi.org/10.1093/zoolinnean/zlac074}{10.1093/zoolinnean/zlac074}}
}

\vspace{4mm}

\textbf{Abstract}. The Phasmida genus \textit{Candovia} comprises nine traditionally recognized species, all endemic to Australia. In this study, \textit{Candovia} diversity is explored through molecular species-delimitation analyses using the \textit{COI\textsubscript{Fol}} gene fragment and phylogenetic inferences leveraging seven additional mitochondrial and nuclear loci. Molecular results were integrated with morphological observations, leading us to confirm the already described species and to the delineation of several new taxa and of the new genus \textit{Paracandovia}. New \textit{Candovia} species from various parts of Queensland and New South Wales are described and illustrated (\textit{C. alata} sp. nov., \textit{C. byfieldensis} sp. nov., \textit{C. dalgleishae} sp. nov., \textit{C. eungellensis} sp. nov., \textit{C. karasi} sp. nov., \textit{C. koensi} sp. nov. and \textit{C. wollumbinensis} sp. nov.). New combinations are proposed and species removed from synonymy with the erection of the new genus \textit{Paracandovia} (\textit{P. cercata} stat. rev., comb. nov., \textit{P. longipes} stat. rev., comb. nov., \textit{P. pallida} comb. nov., \textit{P. peridromes} comb. nov., \textit{P. tenera} stat. rev., comb. nov.). Phylogenetic analyses suggest that the egg capitulum may have independently evolved multiple times throughout the evolutionary history of these insects. Furthermore, two newly described species represent the first taxa with fully developed wings in this previously considered apterous clade.

\clearpage

%----------------------------------------------------------------------------------

{
\setstretch{1.0}
\section*{\LARGE{Comparative genomics of \textit{Hox} and \textit{ParaHox} \\ genes among major lineages of Branchiopoda with emphasis on tadpole shrimps.}}

\vspace{4mm}

\noindent{\large{Filippo Nicolini\textsuperscript{1,2}, Jacopo Martelossi\textsuperscript{1}, Giobbe Forni\textsuperscript{3}, \\ Castrense Savojardo\textsuperscript{4}, Barbara Mantovani\textsuperscript{1}, Andrea Luchetti\textsuperscript{1}}}

\vspace{4mm}

\noindent{\textsuperscript{1}\textit{Department of Biological, Geological and Environmental Sciences, University of Bologna, Bologna, Italy}.}

\noindent{\textsuperscript{2}\textit{Fano Marine Center, Fano (PU), Italy}.}

\noindent{\textsuperscript{3}\textit{Department of Agricultural and Environmental Sciences, University of Milan, Milan, Italy}.}

\noindent{\textsuperscript{4}\textit{Department of Pharmacy and Biotechnology, University of Bologna, Bologna, Italy}.}

\vspace{4mm}

\noindent{\textbf{Published in}: 2023, \textit{Frontiers in Ecology and Evolution}, 11:1046960. \hfill \\ \href{https://doi.org/10.3389/fevo.2023.1046960}{10.3389/fevo.2023.1046960}}
}

\vspace{4mm}

\textbf{Abstract}. \textit{Hox} and \textit{ParaHox} genes (HPHGs) are key developmental genes that pattern regional identity along the anterior–posterior body axis of most animals. Here, we identified HPHGs in tadpole shrimps (Pancrustacea, Branchiopoda, Notostraca), an iconic example of the so-called “living fossils” and performed a comparative genomics analysis of HPHGs and the \textit{Hox} cluster among major branchiopod lineages. Notostraca possess the entire \textit{Hox} complement, and the \textit{Hox} cluster seems to be split into two different subclusters, although we were not able to support this finding with chromosome-level assemblies. However, the genomic structure of \textit{Hox} genes in Notostraca appears more derived than that of \textit{Daphnia} spp., which instead retains the plesiomorphic condition of a single compact cluster. Spinicaudata and \textit{Artemia franciscana} show instead a \textit{Hox} cluster subdivided across two or more genomic scaffolds with some orthologs either duplicated or missing. Yet, branchiopod HPHGs are similar among the various clades in terms of both intron length and number, as well as in their pattern of molecular evolution. Sequence substitution rates are in fact generally similar for most of the branchiopod \textit{Hox} genes and the few differences we found cannot be traced back to natural selection, as they are not associated with any signals of diversifying selection or substantial switches in selective modes. Altogether, these findings do not support a significant stasis in the Notostraca \textit{Hox} cluster and further confirm how morphological evolution is not tightly associated with genome dynamics.

\clearpage

%----------------------------------------------------------------------------------

{
\setstretch{1.0}
\section*{\LARGE{Multiple and diversified transposon lineages contribute to early and recent bivalve genome evolution.}}

\vspace{4mm}

\noindent{\large{Jacopo Martelossi\textsuperscript{1}, Filippo Nicolini\textsuperscript{1,2}, Simone Subacchi\textsuperscript{1}, Daniela \\ Pasquale\textsuperscript{1}, Fabrizio Ghiselli\textsuperscript{1}, Andrea Luchetti\textsuperscript{1}}}

\vspace{4mm}

\noindent{\textsuperscript{1}\textit{Department of Biological, Geological and Environmental Sciences, University of Bologna, Bologna, Italy}.}

\noindent{\textsuperscript{2}\textit{Fano Marine Center, Fano (PU), Italy}.}

\vspace{4mm}

\noindent{\textbf{Published in}: 2023, \textit{BMC Biology}, 21:145. \href{https://doi.org/10.1186/s12915-023-01632-z}{10.1186/s12915-023-01632-z}}
}

\vspace{4mm}

\textbf{Abstract}. \textbf{Background}. Transposable elements (TEs) can represent one of the major sources of genomic variation across eukaryotes, providing novel raw materials for species diversification and innovation. While considerable effort has been made to study their evolutionary dynamics across multiple animal clades, molluscs represent a substantially understudied phylum. Here, we take advantage of the recent increase in mollusc genomic resources and adopt an automated TE annotation pipeline combined with a phylogenetic tree-based classification, as well as extensive manual curation efforts, to characterize TE repertories across 27 bivalve genomes with a particular emphasis on DDE/D class II elements, long interspersed nuclear elements (LINEs), and their evolutionary dynamics. \textbf{Results}. We found class I elements as highly dominant in bivalve genomes, with LINE elements, despite less represented in terms of copy number per genome, being the most common retroposon group covering up to 10\% of their genome. We mined 86,488 reverse transcriptases (RVT) containing LINE coming from 12 clades distributed across all known superfamilies and 14,275 class II DDE/D-containing transposons coming from 16 distinct superfamilies. We uncovered a previously underestimated rich and diverse bivalve ancestral transposon complement that could be traced back to their most recent common ancestor that lived about 500 Mya. Moreover, we identified multiple instances of lineage-specific emergence and loss of different LINEs and DDE/D lineages with the interesting cases of CR1-Zenon, Proto2, RTE-X, and Academ elements that underwent a bivalve-specific amplification likely associated with their diversification. Finally, we found that this LINE diversity is maintained in extant species by an equally diverse set of long-living and potentially active elements, as suggested by their evolutionary history and transcription profiles in both male and female gonads. \textbf{Conclusions}. We found that bivalves host an exceptional diversity of transposons compared to other molluscs. Their LINE complement could mainly follow a “stealth drivers” model of evolution where multiple and diversified families are able to survive and co-exist for a long period of time in the host genome, potentially shaping both recent and early phases of bivalve genome evolution and diversification. Overall, we provide not only the first comparative study of TE evolutionary dynamics in a large but understudied phylum such as Mollusca, but also a reference library for ORF-containing class II DDE/D and LINE elements, which represents an important genomic resource for their identification and characterization in novel genomes.

\clearpage

%----------------------------------------------------------------------------------

{
\setstretch{1.0}
\section*{\LARGE{Towards a time-tree solution for Branchiopoda diversification: a jackknife assessment of fossil age priors.}}

\vspace{4mm}

\noindent{\large{Niccolò Righetti\textsuperscript{1}*, Filippo Nicolini\textsuperscript{2}*, Giobbe Forni\textsuperscript{2}, Andrea Luchetti\textsuperscript{2}}}

\vspace{4mm}

\noindent{\textsuperscript{1}\textit{Laboratoire de Biologie Computationnelle et Quantitative (LCQB), Sorbonne Université, CNRS, IBPS, UMR7238, Paris, France}.}

\noindent{\textsuperscript{2}\textit{Department of Biological, Geological and Environmental Sciences, University of Bologna, Bologna, Italy}.}

\noindent{* the authors equally contributed to this work.}

\vspace{4mm}

\noindent{\textbf{\textit{Submitted for peer-review.}}}
}

\vspace{4mm}

\textbf{Abstract}. An understanding of Branchiopoda's evolutionary history is crucial for a comprehensive knowledge of the Pancrustacea tree of life, given their close evolutionary relationship with Hexapoda. Despite significant advances in molecular and morphological phylogenetics that have resolved much of the branchiopod backbone topology, a reliable temporal framework remains elusive. Key challenges include a sparse fossil record, long-term morphological stasis, and past topological inconsistencies. Leveraging a Bayesian Inference approach and the most extensive phylogenomic dataset for branchiopod to date, encompassing 46 species and over 130 genes, we inferred a time-calibrated phylogenetic tree. Furthermore, to strengthen the confidence in our divergence times estimation, we assessed the impact of age priors, topological uncertainties, and gene trees which are discordant from the species trees. Our results are largely consistent with the fossil record and with previous studies, indicating that Branchiopoda originated between 400 and 500 million years ago, and the orders of large branchiopods diversified during the Mesozoic. Concerning Cladocera, results remain problematic, with a sharper uncertainty in the diversification time with respect to the fossil record. Though, the jackknife resampling of fossils and the other sensitivity analyses proved our calibration method to be robust, suggesting that the difficulties in obtaining a paleontological-consistent time tree may be hindered by the variability in branchiopod substitution rates and topological instability within certain clades.

% \end{document}